app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/ContributorsActivity.java

```diff
@@ -51,8 +54,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.item_repository:
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/DroidKaigi/conference-app-2017"));
-                startActivity(intent);
+                navigator.navigateToWebPage("https://github.com/DroidKaigi/conference-app-2017");
```

Comment:
My sample is here...
https://github.com/ogapants/conference-app-2017/commit/2ef017f24a9ed63d8162acdff56503b2de630eb2